xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -233,3 +236,18 @@ std::unique_ptr<CVideoSync> CWinSystemX11GLContext::GetVideoSync(void *clock)
   }
   return pVSync;
 }
+
+void* CWinSystemX11GLContext::GetVaDisplay()
+{
+#if defined(HAVE_LIBVA)
+  if (dynamic_cast<CGLContextEGL*>(m_pGLContext))
```

Comment:
this is not required. without an EGL context you won't call VAAPI::CDecoder::CheckCaps. without this call vaapi won't get enabled and won't call this method